res/cmake/def.cmake

```diff
@@ -10,6 +10,7 @@ if (phare_configurator)
   execute_process(
     COMMAND ./tools/config/cmake.sh
     WORKING_DIRECTORY ${PHARE_PROJECT_DIR}
+    COMMAND_ERROR_IS_FATAL ANY
```

Comment:
this supposedly needed cmake 3.19 so hence the version upgrade
2.5 years since release I think is reasonable, for cmake 3.20.1